src/patch.js

```diff
@@ -55,7 +55,7 @@ function setElementProp(element, name, value, oldValue) {
   if (name === "key") {
   } else if (name === "style") {
     for (var name in merge(oldValue, (value = value || {}))) {
-      element.style[name] = value[name] || ""
+      element.style[name] = value[name] != null ? value[name] : ""
```

Comment:
@acstll Could we get away with?

```js
element.style[name]+=""
```

Reply:
With `element.style[name]+=""` tests fail.

`!= null` checks against `null` and `undefined`, so `0` is valid. Anything else will be already coerced into a string by the browser.

If I'm correct, this change will keep things intact and only allow `0` to be valid and not being swallowed as falsy. Everything else stays the same.

Reply:
Sorry, I wasn't thinking well! 😅